### PR TITLE
Image data extraction

### PR DESCRIPTION
**Pull request by ZCX031116**

### CODE REVIEW

**Comment by ZCX031116** [comment]
Database structure + Upload page display
Update picture Database from single quiz structure to Image + lookups + quiz structure

Convert pictures to .jpeg when storing (making it convenient to display on webpage)

Convert .heic with heicConvert and others with sharp

Using path to get the extension of file and determine which converter to use

Add app.get in Server.js to access pictures from database

Add displaying uploaded picture feature in ImageUpload.js, to confirm the pics are successfully uploaded

**Comment by ZCX031116** [comment]
**postImage + getImage**
* Separate the function of getting image to display from server.js (app.get()) to getImage() in controller.js and image.js router

* Separate the function of posting single image to databse from createGame() to a function named postImage in controller.js. Based on property of this function, it can only be used when creating a game.

File: src/backend/.DS_Store
Comment (mrElnekave) [inline src/backend/.DS_Store:1]: Check if needs to be tracked on github

File: src/backend/models/Lookup.js [hunk 1]
```diff
@@ -8,19 +8,27 @@ const mongoose = require('mongoose');
 
 const Schema = mongoose.Schema;
 
+// const lookupSchema = new Schema({
```
Comment (mrElnekave) [inline src/backend/models/Lookup.js:11]: Remove the comments

File: src/backend/routes/game_info.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
-const { createGame, deleteGame, getGame, getGames, patchGame } = require('../controllers/controller');
 const express = require('express');
+const multer = require('multer');
+const { createGame, deleteGame, getGame, getGames, patchGame } = require('../controllers/controller');
 
-// router is a mini express app
 const router = express.Router();
-
+const upload = multer({ storage: multer.memoryStorage() });
```
Comment (mrElnekave) [inline src/backend/routes/game_info.js:6]:
```suggestion
const upload = multer({ storage: multer.memoryStorage() }); // way to upload binary images
```

**Comment by mrElnekave** [comment]
Checked that everything runs.

**Pull request merged**